### PR TITLE
Auto update Microsoft.NETCore.App package version

### PR DESCRIPTION
**Pull request by joperezr**
cc: @weshaggard @dagood 

Small change that will update the netcoreapp package automatically with the auto-PRs

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -116,12 +122,16 @@
...
       <PackageId>TestILC.amd64ret</PackageId>
     </XmlUpdateStep>
-    <XmlUpdateStep Include="Sni">
+    <XmlUpdateStep Include="CoreSetup">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>SniPackageVersion</ElementName>
-      <!-- Pick any of the SNI runtime packages. All the runtime packages of sni will have the same version -->
       <PackageId>runtime.win7-x64.runtime.native.System.Data.SqlClient.sni</PackageId>
     </XmlUpdateStep>
+    <XmlUpdateStep Include="Sni">
```
Comment (dagood) [inline dependencies.props]: The name on this step and the one for `SniPackageVersion` seem swapped.

File: dependencies.props [hunk 2]
```diff
@@ -28,6 +29,7 @@
     <ProjectNTfsTestILCPackageVersion>1.0.0-beta-25308-00</ProjectNTfsTestILCPackageVersion>
     <NETStandardPackageVersion>2.0.0-preview2-25308-01</NETStandardPackageVersion>
     <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
+    <CoreSetupPackageVersion>2.0.0-preview2-002093-00</CoreSetupPackageVersion>
```
Comment (dagood) [inline dependencies.props]: I slightly prefer giving these properties explicit names, like `MicrosoftNETCoreAppPackageVersion` in this case. Precise and avoids overlap if we ever depend on another Core-Setup package in the future with a different version. Other props here don't follow the convention, up to you.

File: dependencies.props (hunk 1 above)
Comment (joperezr) [inline dependencies.props]: Good catch, will fix

File: dependencies.props (hunk 2 above)
Comment (joperezr) [inline dependencies.props]: Makes sense, I'll make the change.

File: dependencies.props [hunk 3]
```diff
@@ -56,6 +58,10 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)coreclr/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
+    <RemoteDependencyBuildInfo Include="MicrosoftNETCoreApp">
```
Comment (dagood) [inline dependencies.props]: The `RemoteDependencyBuildInfo`s actually are one per repo, so this should stay as `CoreSetup` (same for the CurrentRef).

File: dependencies.props [hunk 4]
```diff
@@ -119,9 +125,13 @@
     <XmlUpdateStep Include="Sni">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>SniPackageVersion</ElementName>
-      <!-- Pick any of the SNI runtime packages. All the runtime packages of sni will have the same version -->
       <PackageId>runtime.win7-x64.runtime.native.System.Data.SqlClient.sni</PackageId>
     </XmlUpdateStep>
+    <XmlUpdateStep Include="MicrosoftNETCoreApp">
```
Comment (dagood) [inline dependencies.props]: The itemspec here should match the `RemoteDependencyBuildInfo` itemspec, `CoreSetup`. Nothing in CoreFX itself depends on this, but it's used in build from source tooling.

Sorry for the cycles on these names. :smile:

**Pull request merged**